skbio/io/format/fasta.py

```diff
@@ -581,6 +581,60 @@
 >>> new_fasta_fh.close()
 >>> new_qual_fh.close()
 
+Reading multi-FASTA Files
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Suppose you have a multi-FASTA file and want to read each sequence into a ``DNA``
+object in a list. We'll be using ``io.StringIO`` to make a mock FASTA file in
+memory.
+
+>>> from io import StringIO
+>>> import skbio
+>>> from skbio.sequence import DNA
+>>> fl = ">seq1 Turkey\n" +\
+...      "AAGCTNGGGCATTTCAGGGTGAGCCCGGGCAATACAGGGTAT\n" +\
+...      ">seq2 Salmo gair\n" +\
+...      "AAGCCTTGGCAGTGCAGGGTGAGCCGTGG\n" +\
+...      "CCGGGCACGGTAT\n" +\
+...      ">seq3 H. Sapiens\n" +\
+...      "ACCGGTTGGCCGTTCAGGGTACAGGTTGGCCGTTCAGGGTAA\n" +\
+...      ">seq4 Chimp\n" +\
+...      "AAACCCTTGCCG\n" +\
+...      "TTACGCTTAAAC\n" +\
+...      "CGAGGCCGGGAC\n" +\
+...      "ACTCAT\n" +\
+...      ">seq5 Gorilla\n" +\
+...      "AAACCCTTGCCGGTACGCTTAAACCATTGCCGGTACGCTTAA\n"
+>>> mock_fl = StringIO(fl)
+>>> res = list(skbio.io.read(mock_fl, format="fasta", constructor=DNA))
```

Comment:
Maybe add a comment above this line, like "The following code will read the sequences into scikit-bio. In practice, `mock_fl` may be replaced with an opened file handle, or the path to the file."